lib/uniffi-zcash/src/hdwallet/extended_key/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod extended_priv_key;
+pub use extended_priv_key::*;
```

Comment:
Granularity I've decided on is a file per interface/object.
Here `extended_key` directory is for https://docs.rs/hdwallet/latest/hdwallet/extended_key/index.html.
And `expended_priv_key` is for https://docs.rs/hdwallet/latest/hdwallet/extended_key/struct.ExtendedPrivKey.html specifically.

We could reduce it to file per module - what do you think @eloylp ?

Reply:
No strong opinion on my side. I think this code structure its more for the developers of the project rather than the final users, who are going to see the same flat structure no matter what we do here.  We already did a great job with the first split, so maybe i would put everything in the `mod.rs` file for the moment.